scripts/supermq.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Copyright (c) Abstract Machines
+# SPDX-License-Identifier: Apache-2.0
+
+###
+# Fetches the latest version of the docker files from the SuperMQ repository.
+###
+
+set -e
+set -o pipefail
+
+REPO_URL=https://github.com/absmach/supermq
+TEMP_DIR="supermq"
+DOCKER_FOLDER="docker"
+DEST_FOLDER="../../docker/supermq-docker"
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+cd "$SCRIPT_DIR" || exit 1
+
+if [ -n "$(git status --porcelain)" ]; then
+    echo "There are uncommitted changes. Please commit or stash them before running this script."
+    exit 1
+fi
+
+cleanup() {
+    rm -rf "$TEMP_DIR" "../$TEMP_DIR"
```

Comment:
Is `"../$TEMP_DIR"` required?

Reply:
I initially included it to clean up incase there was an error in sparse checkout or in moving the contents of the folder but i dont think it will be required.